types/underscore/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "exports": {
```

Comment:
This is hugely simplified compared to the export map [underscore actually provides](https://github.com/jashkenas/underscore/blob/master/package.json#L22), however `@types/underscore` currently has no definitions for... any? of the submodules exposed, so.... yeah, this is all we get.

Reply:
In that case... shouldn't we mirror the entire exports hierarchy of Underscore, rather than only the UMD and Node-esm flavors? This seems somewhat arbitrary.

Reply:
I'm not going to add definitions for all the other modules `underscore` exposes - but hey, I won't stop someone else from putting in the work. I'm just here to make the main esm entrypoint work.